test/mocks/customers_list.js

```diff
@@ -18,7 +18,7 @@ nock('https://api.omise.co')
       email: 'john.doe@example.com',
       description: 'John Doe (id: 30)',
       created: '2015-02-03T05:28:15Z',
-      cards: [Object]
+      cards: [Object],
```

Comment:
just saw we have this in the code. hopefully the tests are still valid 😂

Reply:
The dangling comma in object literal is valid. It's up to our agreement.

Currently, the rule that force dangling comma is enabled by Google's eslint configuration, but we can override it.